cmd/disk-cache.go

```diff
@@ -230,7 +230,7 @@ func (c cacheObjects) GetObject(ctx context.Context, bucket, object string, star
 		return err
 	}
 	go func() {
-		if err = GetObjectFn(ctx, bucket, object, 0, objInfo.Size, io.MultiWriter(writer, pipeWriter), etag); err != nil {
```

Comment:
These changes are not related to the changes mentioned by this PR, would it be possible to send this separately?

Reply:
Yep, sure.